CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.34.0
+------------------------------
+*March 8, 2018*
```

Comment:
Is this just a super pessimistic estimate? 🤣 ⌛️

Reply:
Perhaps I should of left it now, lol